### PR TITLE
Remove unneeded #pragma's

### PR DESCRIPTION
**Pull request by kupsch**
- Remove once and interface pragmas
- Replace use of "external/stdint-win.h" and "external/inttypes.h" with
  <stdint.h> and <inttypes.h>
- Remove unused files containing pragmas

### CODE REVIEW

File: proccontrol/src/procset.C [hunk 1]
```diff
@@ -45,10 +45,6 @@
 #include <map>
 #include <algorithm>
 
-#if defined(os_windows)
```
Comment (hainest) [inline proccontrol/src/procset.C:48]: Did we need to include stdint.h and inttype.h here, or are they not used in this file?

**Review by hainest — COMMENTED** [review]
common/src/refCounter.h can go away completely as it's never used. I would do that separately.

File: proccontrol/src/procset.C (hunk 1 above)
Comment (kupsch) [reply proccontrol/src/procset.C:48]: They are not necessary for this file.

**Review by hainest — APPROVED** [review]
LGTM.

**Pull request merged**